chore: Re-generated to pick up changes in the API or client library generator.

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/56d96910-3888-4012-8dcf-111aa1013a5b/targets

- [ ] To automatically regenerate this PR, check this box. (May take up to 24 hours.)

PiperOrigin-RevId: 373649163
Source-Link: https://github.com/googleapis/googleapis/commit/7e1b14e6c7a9ab96d2db7e4a131981f162446d34
PiperOrigin-RevId: 373400747
Source-Link: https://github.com/googleapis/googleapis/commit/162641cfe5573c648df679a6dd30385650a08704
PiperOrigin-RevId: 372197450
Source-Link: https://github.com/googleapis/googleapis/commit/83a7e1c8c2f7421ded45ed323eb1fda99ef5ea46
PiperOrigin-RevId: 371362703
Source-Link: https://github.com/googleapis/googleapis/commit/5a04154e7c7c0e98e0e4085f6e2c67bd5bff6ff8
PiperOrigin-RevId: 370989216
Source-Link: https://github.com/googleapis/googleapis/commit/4e825559e5ab242c4d1aafca19c83c4d1bac743e
PiperOrigin-RevId: 370926454
Source-Link: https://github.com/googleapis/googleapis/commit/382ed8de075e1ddc9baa4ebf8dbc5b2c257a77b8
PiperOrigin-RevId: 370525906
Source-Link: https://github.com/googleapis/googleapis/commit/60e129d0672a1be2c70b41bf76aadc7ad1b1ca0f
PiperOrigin-RevId: 364411656
Source-Link: https://github.com/googleapis/googleapis/commit/149a3a84c29c9b8189576c7442ccb6dcf6a8f95b
PiperOrigin-RevId: 364356707
Source-Link: https://github.com/googleapis/googleapis/commit/5ef92c3f11395a59f55ced317423b992e61d5cfe
Source-Link: https://github.com/googleapis/synthtool/commit/1efbe51eaa8b153bdef51f8715ff75a71f409004
PiperOrigin-RevId: 361662015
Source-Link: https://github.com/googleapis/googleapis/commit/28a591963253d52ce3a25a918cafbdd9928de8cf
PiperOrigin-RevId: 359562873
Source-Link: https://github.com/googleapis/googleapis/commit/07932bb995e7dc91b43620ea8402c6668c7d102c
PiperOrigin-RevId: 355923884
Source-Link: https://github.com/googleapis/googleapis/commit/5e3dacee19405529b841b53797df799c2383536c
PiperOrigin-RevId: 354996675
Source-Link: https://github.com/googleapis/googleapis/commit/20712b8fe95001b312f62c6c5f33e3e3ec92cfaf

dinagraves commented:
`AttributeError: type object 'GrafeasClient' has no attribute 'DEFAULT_ENDPOINT'`

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.40.5

PiperOrigin-RevId: 354996675

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Feb 1 12:11:49 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 20712b8fe95001b312f62c6c5f33e3e3ec92cfaf
Source-Link: https://github.com/googleapis/googleapis/commit/20712b8fe95001b312f62c6c5f33e3e3ec92cfaf

yoshi-automation committed:
chore: update gapic-generator-python

PiperOrigin-RevId: 355923884

Source-Author: Google APIs <noreply@google.com>
Source-Date: Fri Feb 5 14:04:52 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 5e3dacee19405529b841b53797df799c2383536c
Source-Link: https://github.com/googleapis/googleapis/commit/5e3dacee19405529b841b53797df799c2383536c

yoshi-automation committed:
chore: update gapic-generator-python to 0.40.11

PiperOrigin-RevId: 359562873

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu Feb 25 10:52:32 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 07932bb995e7dc91b43620ea8402c6668c7d102c
Source-Link: https://github.com/googleapis/googleapis/commit/07932bb995e7dc91b43620ea8402c6668c7d102c

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.42.2

PiperOrigin-RevId: 361662015

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 8 14:47:18 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 28a591963253d52ce3a25a918cafbdd9928de8cf
Source-Link: https://github.com/googleapis/googleapis/commit/28a591963253d52ce3a25a918cafbdd9928de8cf

yoshi-automation committed:
fix: lint all ts/js

Source-Author: Benjamin E. Coe <bencoe@google.com>
Source-Date: Wed Mar 17 08:53:10 2021 -0700
Source-Repo: googleapis/synthtool
Source-Sha: 1efbe51eaa8b153bdef51f8715ff75a71f409004
Source-Link: https://github.com/googleapis/synthtool/commit/1efbe51eaa8b153bdef51f8715ff75a71f409004

yoshi-automation committed:
docs: fix link for version and environment in participant doc. fix: Add all missing regional HTTP bindings in DF API.

PiperOrigin-RevId: 364356707

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 22 10:50:15 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 5ef92c3f11395a59f55ced317423b992e61d5cfe
Source-Link: https://github.com/googleapis/googleapis/commit/5ef92c3f11395a59f55ced317423b992e61d5cfe

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.43.1

PiperOrigin-RevId: 364411656

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 22 14:40:22 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 149a3a84c29c9b8189576c7442ccb6dcf6a8f95b
Source-Link: https://github.com/googleapis/googleapis/commit/149a3a84c29c9b8189576c7442ccb6dcf6a8f95b

yoshi-automation committed:
chore: upgrade to gapic-generator-python 0.44.0

chore: add GAPIC metadata
feat: add support for self-signed JWT
PiperOrigin-RevId: 370525906

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Apr 26 13:12:26 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 60e129d0672a1be2c70b41bf76aadc7ad1b1ca0f
Source-Link: https://github.com/googleapis/googleapis/commit/60e129d0672a1be2c70b41bf76aadc7ad1b1ca0f

yoshi-automation committed:
chore: upgrade gapic-generator python to 0.44.1

PiperOrigin-RevId: 370926454

Source-Author: Google APIs <noreply@google.com>
Source-Date: Wed Apr 28 09:58:07 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 382ed8de075e1ddc9baa4ebf8dbc5b2c257a77b8
Source-Link: https://github.com/googleapis/googleapis/commit/382ed8de075e1ddc9baa4ebf8dbc5b2c257a77b8

yoshi-automation committed:
chore: upgrade gapic-generator python to 0.44.1

PiperOrigin-RevId: 370989216

Source-Author: Google APIs <noreply@google.com>
Source-Date: Wed Apr 28 14:42:12 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 4e825559e5ab242c4d1aafca19c83c4d1bac743e
Source-Link: https://github.com/googleapis/googleapis/commit/4e825559e5ab242c4d1aafca19c83c4d1bac743e

yoshi-automation committed:
chore: revert to gapic-generator-python 0.43.3

PiperOrigin-RevId: 371362703

Source-Author: Google APIs <noreply@google.com>
Source-Date: Fri Apr 30 10:44:40 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 5a04154e7c7c0e98e0e4085f6e2c67bd5bff6ff8
Source-Link: https://github.com/googleapis/googleapis/commit/5a04154e7c7c0e98e0e4085f6e2c67bd5bff6ff8

yoshi-automation committed:
fix: add async client to %name_%version/init.py chore: add autogenerated snippets chore: remove auth, policy, and options from the reserved names list feat: support self-signed JWT flow for service accounts chore: enable GAPIC metadata generation chore: sort subpackages in %namespace/%name/init.py

PiperOrigin-RevId: 372197450

Source-Author: Google APIs <noreply@google.com>
Source-Date: Wed May 5 13:39:02 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 83a7e1c8c2f7421ded45ed323eb1fda99ef5ea46
Source-Link: https://github.com/googleapis/googleapis/commit/83a7e1c8c2f7421ded45ed323eb1fda99ef5ea46

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.46.1

PiperOrigin-RevId: 373400747

Source-Author: Google APIs <noreply@google.com>
Source-Date: Wed May 12 10:34:35 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 162641cfe5573c648df679a6dd30385650a08704
Source-Link: https://github.com/googleapis/googleapis/commit/162641cfe5573c648df679a6dd30385650a08704

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.46.3

PiperOrigin-RevId: 373649163

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu May 13 13:40:36 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 7e1b14e6c7a9ab96d2db7e4a131981f162446d34
Source-Link: https://github.com/googleapis/googleapis/commit/7e1b14e6c7a9ab96d2db7e4a131981f162446d34

parthea commented:
Superseded by #75
